.github/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+x. AIRAC (2407) - Lashenden NDB removed - thanks to @Bucklerz (Reece Buckley)
```

Comment:
```suggestion
x. AIRAC (2407) - Removed Lashenden NDB (LSH) - thanks to @Bucklerz (Reece Buckley)
```
Per the [Style Guide for Changelog Entries](https://github.com/VATSIM-UK/UK-Sector-File/blob/main/.github/Style%20Guide.md#changelog-entries--), the summary should start with the past tense verb. I think for searchability also worth having the NDB identifier in there.

The PR title should start with the imperitive tense verb (I have changed this for you).